src/stylesheets/_variables.scss

```diff
@@ -1,6 +1,7 @@
 // Colors
 $black: rgb(0, 0, 0);
```

Comment:
You should prefix all your custom variables with a names so you avoid clashing with any third party module variables. 

So `$white` can be `$os-white` (for "online shopping")